src/components/manager/projects/Project.js

```diff
@@ -75,25 +78,34 @@ const Project = ({
                 ...projectState,
                 ...value,
                 data_use: simpleDeepCopy(value.data_use || INITIAL_DATA_USE_VALUE),
+                discovery: newDiscoveryContent || value.discovery,
             });
         }
-    }, [value]);
+    }, [value, newDiscoveryContent]);
+
+
 
     const handleSave = useCallback(() => {
-        const form = editingForm.current;
-        if (!form) return;
-        form.validateFields().then((values) => {
+        editingForm.validateFields().then((values) => {
             // Don't save datasets since it's a related set.
+            console.log(values);
             onSave({
                 identifier: projectState.identifier,
                 title: values.title || projectState.title,
                 description: values.description || projectState.description,
                 data_use: values.data_use || projectState.data_use,
+                discovery: newDiscoveryContent || values.discovery,
             });
+            editingForm.resetFields();
```

Comment:
might want to await onsave, or reset form on open instead of on save to prevent the flicker of the form back to the reset values